openhtf/io/output/mfg_inspector.py

```diff
@@ -71,6 +72,7 @@
 # measurement.
 MAX_PARAMS_PER_MEASUREMENT = 100
 
+ERROR_MESSAGE = 'Upload to mfg-inspector failed!'
```

Comment:
i don't think this really needs to be declared up top or configurable, it can just be used inline directly.